subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt

```diff
@@ -103,6 +132,38 @@ public class InstrumentationTestsPlugin : Plugin<Project> {
                 )
             )
         }
+
+        extension.environmentsContainer.registerFactory(KubernetesViaCredentials::class.java) {
+            project.objects.newInstance(KubernetesViaCredentials::class.java, it)
+        }
+        extension.environmentsContainer.registerFactory(KubernetesViaContext::class.java) {
+            project.objects.newInstance(KubernetesViaContext::class.java, it)
+        }
+        extension.environmentsContainer.registerFactory(LocalAdb::class.java) {
+            project.objects.newInstance(LocalAdb::class.java, it)
+        }
+
+        extension.environmentsContainer.register<LocalAdb>("local")
+
         return extension
     }
+
+    /**
+     * todo remove (registered for backward compatibility)
+     */
+    private fun registerDefaultEnvironment(
```

Comment:
When do you plan to remove `default` env?

Reply:
Right after CISteps removal, because almost every build uses instrumentation tasks, which uses default environment for now; so in a couple of days